src/Http/Http.Features/src/Microsoft.AspNetCore.Http.Features.csproj

```
@@ -10,10 +10,14 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-
-  <ItemGroup>
+   <ItemGroup>
```

Agree w/ the newline removal but not the excess indentation
```suggestion
 <ItemGroup>
```

---

This was the trickiest part to resolve, for some reason my computer showed 2 spaces but on GitHub it was clearly 3, thanks